apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -122,6 +123,10 @@ type TempoStackSpec struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Observability"
 	Observability ObservabilitySpec `json:"observability,omitempty"`
+
+	// +required
+	// +kubebuilder:pruning:PreserveUnknownFields
+	ExtraConfig map[string]apiextensionsv1.JSON `json:"extraConfig,omitempty"`
```

Comment:
We should document here the support level for this option. Maybe we should as well consider submitting a warning in the webhook if it is used.

Reply:
The warning is actually a good idea